### PR TITLE
Remove TODOs related to Crystal 0.22

### PR DESCRIPTION
**Pull request by Sija**
One `TODO` less.

### CODE REVIEW

**Comment by RX14** [comment]
Personally I think we should just remove the `skip_file`s on `src/crystal/system`. Keep the logic very very clear and simple in the `src/crystal/system/foo.cr` top-level file.

File: src/crystal/system/unix/urandom.cr [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# TODO: replace with `flag?(:unix) && !flag?(:openbsd) && !flag?(:linux)` after crystal > 0.22.0 is released
-{% skip_file() if flag?(:openbsd) && flag?(:linux) %}
+{% skip_file if flag?(:unix) && !flag?(:openbsd) && !flag?(:linux) %}
```
Comment (lbguilherme) [inline src/crystal/system/unix/urandom.cr]: I wonder why this checks to only skip on `unix` that isn't openbsd or linux if this file would only be `require`d for unix already? This is kind of telling "dont skip if it is windows", which doesn't make sense.

Comment (Sija) [reply src/crystal/system/unix/urandom.cr]: Yep, there's sth off here, it erroneously skips file for OS X.

**Comment by Sija** [comment]
Rebased on `master`.

**Pull request merged**